### ISSUE TITLE
KDE Plasma on Wayland isn't supported

### ISSUE THREAD

**Issue by moceap** [phw/peek#935]
KDE Plasma on Wayland isn't supported

- With ffmpeg backend: Black records
- With gnome-shell backend: 

Failed to initialize recorder: GDBus.Error:org.freedesktop.DBus.Error.NoReply: Remote peer disconnected

**Comment by bjconlan** [comment]
Yeah, I'm generally seeing the same issue (suspect wayland) this time on fedora 35 (using gnome 41)

**Comment by benjamimgois** [comment]
same here, recording on plasma-wayland results in a black screen, but the mouse cursor shows up.

**Comment by alvaro893** [comment]
Same. Isn't there any workaround for this? I'm bummed out that I have to renounce to Peek because of Wayland

**Comment by shenlebantongying** [comment]
I just learned how to capture GIF/shortvideos using OBS studio. OBS studio is pretty simple and fast. There are tons of related tutorials. This is a workaround I think.

**Comment by haizaar** [comment]
The problem is still here. Tried the below with no luck:
```
$ peek --backend=ffmpeg
Gtk-Message: 21:33:39.418: Failed to load module "appmenu-gtk-module"
Requested screen recording backend ffmpeg
```

KDE 5.25.3 on NixOS
peek-1.5.1
